src/LDNS.xs

```diff
@@ -1223,6 +1289,38 @@ packet_new_from_wireformat(class,buf)
     OUTPUT:
         RETVAL
 
+#
+# Function: set_edns_present
+# --------------------------
+# Set edns_present property of Packet object
+#
+# returns: a boolean (true)
```

Comment:
Does this always return true? What's the point of that?

Also, why not combine set_edns_present and unset_edns_present into one method? Maybe even extend it to be a conventional getter/setter method?

Reply:
The way this flag is implemented in Zonemaster::LDNS ans ldns itself is different from other flags. That's why I prefered to use a non ambiguous way to set/unset this flag and be 100% sure it could only be false/true.
The best way would be to modify ldns itself but it was out of scope.

Reply:
But the return value. Is it necessary to call ldns_pkt_edns() just to calculate true? Wouldn't it be simply more efficient to literally returning true? Or even having it return void - a less confusing alternative to constant return values.

Reply:
Ok, it has been lightened.